src/components/Drawer/docs/Drawer.stories.mdx

```diff
@@ -0,0 +1,115 @@
+import { ArgsTable, Meta, Canvas, Story } from '@storybook/addon-docs';
+import { FigmaImage, FigmaIframe, FigmaLink, Use } from '../../../docs';
+
+import Drawer from '../';
+
+import * as Stories from '../Drawer.stories';
+
+import tokens from '../../../tokens';
+
+<Meta
+	title="Components/Drawer"
+	component={Drawer}
+	decorators={[
+		storyFn => (
+			<div
+				style={{
+					display: 'flex',
+					height: '40vh',
+					border: `1px solid ${tokens.colors.gray[400]}`,
+					overflow: 'hidden',
+				}}
+			>
+				<div style={{ flexShrink: 0, flexBasis: '100%' }}></div>
+				<div style={{ position: 'relative', flexShrink: 0, flexBasis: '40%' }}>{storyFn()}</div>
+			</div>
+		),
+	]}
+/>
+
+# Drawer
+
+Drawer slides into the screen from the right side and is placed visually over the content of the page.
+
+It aims to display extra information while keeping the navigation enabled on the main content.
+
+## Zoning
+
+<FigmaImage
+	src="https://www.figma.com/file/qc8oUSyVJM67gSsbTShftt/%F0%9F%93%90-Zoning?node-id=744%3A263"
+	alt="zoning image for drawers"
+/>
+
+## Style
+
+<FigmaIframe
+	light="https://www.figma.com/file/x0RxG7E0CzVWDycXtNgq7H/Drawer?node-id=1%3A83"
+	dark="https://www.figma.com/file/x0RxG7E0CzVWDycXtNgq7H/Drawer?node-id=1%3A105"
+/>
+
+### Variations
+
+<Story story={Stories.Simple} />
+
+## States
+
+Drawers are either visible or hidden.
+
+## Content
+
+Drawers are “optional” content living on top of primary content. They represent 1/3 of the content, prefer extra or additional information. Their content should be efficient, short and tailored for the limited layout space available.
+
+If you need more space, use [full-page layout]().
```

Comment:
```suggestion
Drawers are “optional” content living on top of primary content. Since they fill only 1/3 of the page's width, prefer extra or additional information. Avoid dense copy and complex forms. Drawer contents should be efficient, short and tailored for the limited layout space available.

If you need more space, use [full-page layout]().
```